include/circt/Dialect/Calyx/CalyxPasses.h

```diff
@@ -31,6 +31,7 @@ std::unique_ptr<mlir::Pass> createResetInsertionPass();
 std::unique_ptr<mlir::Pass> createGroupInvariantCodeMotionPass();
 std::unique_ptr<mlir::Pass> createAffineParallelUnrollPass();
 std::unique_ptr<mlir::Pass> createAffineToSCFPass();
+std::unique_ptr<mlir::Pass> createAffinePloopUnparallelizePass();
```

Comment:
Nit: why is there a `P`? aren't we just creating an `AffineFor`?

Reply:
I made `Ploop` short for `ParallelLoop`, so `createAffineParallelLoopUnparallelizePass`